ts/features/zendesk/saga/networking/handleGetTotalNewResponses.ts

```diff
@@ -0,0 +1,154 @@
+import {
+  call,
+  fork,
+  put,
+  race,
+  select,
+  take,
+  delay
+} from "typed-redux-saga/macro";
+import { isActionOf } from "typesafe-actions";
+import { getError } from "../../../../utils/errors";
+
+import {
+  AnonymousIdentity,
+  getIdentityByToken,
+  getTotalNewResponses,
+  unreadTicketsCountRefreshRate,
+  initSupportAssistance,
+  JwtIdentity,
+  setUserIdentity,
+  ZendeskAppConfig,
+  zendeskDefaultAnonymousConfig,
+  zendeskDefaultJwtConfig,
+  unreadTicketsCountRefreshRateWhileSupportIsOpen
+} from "../../../../utils/supportAssistance";
+import { zendeskTokenSelector } from "../../../../store/reducers/authentication";
+import {
+  logoutRequest,
+  sessionExpired,
+  sessionInformationLoadSuccess,
+  sessionInvalid
+} from "../../../../store/actions/authentication";
+import {
+  zendeskGetTotalNewResponses,
+  zendeskRequestTicketNumber,
+  zendeskSupportOpened
+} from "../../store/actions";
+import { isTestEnv } from "../../../../utils/environment";
+import {
+  versionInfoLoadFailure,
+  versionInfoLoadSuccess
+} from "../../../../common/versionInfo/store/actions/versionInfo";
+import { backendStatusLoadSuccess } from "../../../../store/actions/backendStatus";
+import { Action } from "../../../../store/actions/types";
+import { SagaCallReturnType } from "../../../../types/utils";
+
+function* setupZendesk() {
+  const zendeskToken: string | undefined = yield* select(zendeskTokenSelector);
+
+  const zendeskConfig: ZendeskAppConfig = zendeskToken
+    ? { ...zendeskDefaultJwtConfig, token: zendeskToken }
+    : zendeskDefaultAnonymousConfig;
+
+  yield* call(initSupportAssistance, zendeskConfig);
+  const zendeskIdentity: JwtIdentity | AnonymousIdentity = yield* call(
+    getIdentityByToken,
+    zendeskToken
+  );
+
+  yield* call(setUserIdentity, zendeskIdentity);
+}
+
+function* getTicketsCount() {
+  yield* call(setupZendesk);
+  // Try to get the total messages of the user
+  yield* put(zendeskRequestTicketNumber.request());
+}
+
+function* getUnreadTicketsCount() {
+  yield* call(setupZendesk);
+  // Try to get the new messages of the user
+  try {
+    const response: SagaCallReturnType<typeof getTotalNewResponses> =
```

Comment:
```suggestion
    const response =
```
I believe we don't need it anymore

Reply:
Thank you, removed here [65cb4b8](https://github.com/pagopa/io-app/pull/3770/commits/65cb4b8c50123e2b2c9f81b2220e392004392a4e)